src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompletePanelViewModel.cs

```diff
@@ -22,15 +22,15 @@
 using RestSharp;
 using System.Windows;
 
-namespace Dynamo.NodeAutoComplete
+namespace Dynamo.NodeAutoComplete.ViewModels
 {
-    public class NodeAutoCompleteViewModel : NotificationObject
+    public class NodeAutoCompletePanelViewModel : NotificationObject
```

Comment:
hmm should we stick with `bar` since it's still part of the view naming, or switch to all use `Panel`, or the `Flyout` we have been internally referring to?

Reply:
So we have one view/model for the flyout~= bar and another view/model for the panel. For now these are two different controls and I guess they should stay that way ? or maybe not ? tbd - we have the task showing results in the panel

Reply:
Ah, my mistake, I missed the fact the panel is for the right side panel area. Please ignore my comment above